### ISSUE TITLE
Big event image alt

### ISSUE THREAD

**Issue by axoxelol1** [axoxelol1/dat257-hasse-andersson#95]
When the facebook session for the image URL runs out the alt text "Image for the event" links to this dead link. There should be some check to see if there is an image and if there isn't then there should be a default image.
